### PR TITLE
removing references to legacy libraries

### PR DESCRIPTION
**Pull request by theflyingcodr**

### CODE REVIEW

File: bip32/example_test.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2014 The btcsuite developers
```
Comment (Tigh-Gherr) [inline bip32/example_test.go:1]: I'm nearly certain we've been deliberately preserving these comments as an acknowledgement to the original codebase

Comment (Tigh-Gherr) [reply bip32/example_test.go:1]: Yeah if you do a `grep -rsn --include=*go Copyright`, you'll see a few different copyrights and development teams mentioned

Comment (theflyingcodr) [reply bip32/example_test.go:1]: ah ok

**Review by Tigh-Gherr — APPROVED** [review]
LGTM!!!

**Pull request merged**